lighthouse-core/test/computed/metrics/cumulative-layout-shift-all-frames-test.js

```diff
@@ -6,8 +6,8 @@
 'use strict';
 
 const CumulativeLayoutShiftAllFrames = require('../../../computed/metrics/cumulative-layout-shift-all-frames.js'); // eslint-disable-line max-len
-const frameMetricsTrace = require('../../fixtures/traces/frame-metrics-m89.json');
-const invalidTrace = require('../../fixtures/traces/progressive-app-m60.json');
+const oldTrace = require('../../fixtures/traces/frame-metrics-m89.json');
```

Comment:
oh c'mon that's just mean M89 isn't even stable yet :P

`missingWeightedClsTrace` perhaps?